### DISCUSSION TITLE
How to update the JobDataMap for a paused job?

### DISCUSSION THREAD

**Discussion by ZLVincent** [quartznet/quartznet#1833] [Ideas]
At present only through GetCurrentlyExecutingJobs method to get to the JobDataMap of currently running Job , but if the state of the Job is Paused, can't modify the JobDataMap of this Job, this can support? I want to update the JobDataMap of  the paused job and retrieve the latest JobDataMap after the Job execution resumes

**Comment by lahma** [comment]
https://github.com/quartznet/quartznet/discussions/1823

**Reply by ZLVincent** [reply]
But 1823 hasn't answered for a long time...
